contracts/LockedAccount.sol

```diff
@@ -210,6 +224,7 @@ contract LockedAccount is AccessControlled, AccessRoles, TimeSource, ReturnsErro
 
     /// migrate single investor
     function migrate()
+        acceptAgreement(msg.sender)
```

Comment:
`migrate()` results in signing agreement in `migration` contract so we should remove it here. migration event is generated anyway

Reply:
ok/